pkg/pillar/scripts/device-steps.sh

```diff
@@ -26,6 +27,9 @@ TPMINFOTEMPFILE=/var/tmp/tpminfo.txt
 echo "$(date -Ins -u) Starting device-steps.sh"
 echo "$(date -Ins -u) EVE version: $(cat /run/eve-release)"
 
+#Recording SMART details to a file
+smartctl -a /dev/sda > $SMART_DETAILS_FILE
```

Comment:
Can't hard-code /dev/sda. Need to determine this from where e.g., /persist is mounted.

Reply:
Removed the hardcoded value and fetching `/persist` mount.